Elevel/Application/Features/TestCommands/CreateTestCommand.cs

```diff
@@ -24,25 +29,227 @@ public class Handler : IRequestHandler<Request, Response>
         {
             private readonly IApplicationDbContext _context;
             private readonly IMapper _mapper;
-            public Handler(IApplicationDbContext context, IMapper mapper)
+            private static Random _rand = new Random();
+            private readonly UserManager<ApplicationUser> _userManager;
+
+            private const int GRAMMAR_TEST_COUNT = 12;
+            private const int AUDITION_TEST_COUNT = 10;
+            public Handler(IApplicationDbContext context, IMapper mapper, UserManager<ApplicationUser> userManager)
             {
                 _context = context;
                 _mapper = mapper;
+                _userManager = userManager;
+
             }
             public async Task<Response> Handle(Request request, CancellationToken cancelationtoken)
             {
+                if (await _userManager.Users.FirstOrDefaultAsync(x => x.Id == request.UserId).ConfigureAwait(false) == null)
+                {
+                    return new Response()
+                    {
+                        Id = null,
+                        Message = "UserId is not found"
+                    };
```

Comment:
In this case instead response I supposed using exception e.g. throw EntityNotFoundException("UserId is not found"). Or you can create your own exception e.g. ValidationException or BusinessException